src/modules/navigation/NavContent.jsx

```diff
@@ -6,9 +6,10 @@ import Circle from 'cozy-ui/transpiled/react/Circle'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import OpenWithIcon from 'cozy-ui/transpiled/react/Icons/Openwith'
 import { NavIcon, NavText } from 'cozy-ui/transpiled/react/Nav'
+import Typography from 'cozy-ui/transpiled/react/Typography'
 import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
 
-const NavContent = ({ icon, badgeContent, label, external }) => {
+const NavContent = ({ icon, badgeContent, label, externalDrive }) => {
```

Comment:
nit: `external` is more general maybe we can use it for other purpose in the futur

Reply:
<img width="528" alt="Capture d’écran 2024-04-16 à 15 37 07" src="https://github.com/cozy/cozy-drive/assets/1107936/bce7939f-088e-41c5-9d85-d711f791b11b">
from https://github.com/cozy/cozy-guidelines/?tab=readme-ov-file#naming-of-functions ;)